assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/hooks/useTicketPriceCalculatorModal.tsx

```diff
@@ -1,24 +1,24 @@
 import { useState, useEffect, useCallback } from 'react';
 import { isEmpty, isNil } from 'ramda';
 
-import {
-	useEditorModal,
-	EditorModal,
-	EntityId,
-	EditorModalCallback,
-} from '../../../../../application/ui/components/layout/eeditorModal';
-import { defaultNewPriceModifier } from '../../../../shared/entities/prices/defaultNewPriceModifier';
-import { sortByPriceOrderIdAsc } from '../../../../shared/entities/prices/predicates/sortingPredicates';
-import { copyPriceFields } from '../../../../shared/entities/prices/predicates/updatePredicates';
-import { copyTicketFields } from '../../../../shared/entities/tickets/predicates/updatePredicates';
-import TicketPriceCalculatorForm from '../TicketPriceCalculatorForm';
-import useDefaultPriceType from '../../../data/queries/priceTypes/useDefaultPriceType';
 import useOnSubmitPrices from './useOnSubmitPrices';
 import useTicketPriceCalculatorFormDecorator from './useTicketPriceCalculatorFormDecorator';
 import useTicketPriceCalculatorFormMutators from './useTicketPriceCalculatorFormMutators';
+import TicketPriceCalculatorForm from '../TicketPriceCalculatorForm';
+import useDefaultPriceType from '../../../data/queries/priceTypes/useDefaultPriceType';
 import useTicketPrices from '../../../data/queries/tickets/useTicketPrices';
 import useTicketItem from '../../../data/queries/tickets/useTicketItem';
 import { Ticket, Price } from '../../../../eventEditor/data/types';
+import { defaultNewPriceModifier } from '../../../../shared/entities/prices/defaultNewPriceModifier';
+import { sortByPriceOrderIdAsc } from '../../../../shared/entities/prices/predicates/sortingPredicates';
+import { copyPriceFields } from '../../../../shared/entities/prices/predicates/updatePredicates';
+import { copyTicketFields } from '../../../../shared/entities/tickets/predicates/updatePredicates';
+import {
+	useEditorModal,
+	EditorModal,
+	EntityId,
+	ModalClose,
+} from '../../../../../application/ui/components/layout/eeditorModal';
```

Comment:
Like stuff being in order :1st_place_medal:

Reply:
but `TicketPriceCalculatorForm` is not in order :)

Reply:
You mean in alphabetical order?

Reply:
Yes, alphabetical order.